### PR TITLE
fix: Added self-contained style for Menu component

### PR DESCRIPTION
**Pull request by SaiShan**
Closes SAP/fundamental-styles#158

This PR contains the self-contained styling for the Menu component.
This PR is part of the larger task of making all components self-contained. #136

### CODE REVIEW

File: test/templates/menu/index.njk [hunk 1]
```diff
@@ -1,80 +1,25 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
 {% from "./component.njk" import menu %}
+{% from "./component.njk" import menuWithAllHeadlines %}
```
Comment (greg-a-smith) [inline test/templates/menu/index.njk]: Not sure about Nunjucks syntax, but is there a way to import both `menu` and `menuWithAllHeadlines` from the file in one line?

File: components/menu.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+$fd-icons-path: "../scss/icons/";
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/components/menu";
+@import "../scss/icons/icon";
```
[the remark this answers is not in the document]
Comment (droshev) [reply components/menu.scss]: @greg-a-smith  you are right! This:
```
{% from "../icon/component.njk" import icon %}
```
is not needed nor used.

File: test/templates/menu/component.njk [hunk 3]
```diff
@@ -1,57 +1,62 @@
 {% import "./../utils.njk" as utils %}
-<!--
-menu:
-    properties={},
-    modifier={ block: [] },
-    state={},
-    aria={}
--->
+{% from "../icon/component.njk" import icon %}
```
[the remark this answers is not in the document]
Comment (droshev) [reply test/templates/menu/component.njk]: done

File: test/templates/menu/index.njk (hunk 1 above)
Comment (droshev) [reply test/templates/menu/index.njk]: done

File: components/menu.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (greg-a-smith) [reply components/menu.scss]: Couple things related to this:

1. The `icons` can be removed from the `/components/menu.scss` file.
1. How about we create a custom playground image class in `layout-shared.css` that can be used as a placeholder?

File: test/templates/menu/component.njk (hunk 3 above)
[the remark this answers is not in the document]
Comment (greg-a-smith) [reply test/templates/menu/component.njk]: I am still seeing the import line for icon.

**Comment by droshev** [comment]
@greg-a-smith i thought i had removed it, but it looks like i didn't.
The change is done now.

**Pull request merged**